packages/backend/src/peripherals/database/shared/types.ts

```diff
@@ -35,10 +35,9 @@ declare module 'knex/types/tables' {
   interface AggregatedReportRow {
     unix_timestamp: Date
     project_id: string
-    tvl_usd: string
-    tvl_eth: string
-    is_daily: boolean
-    is_six_hourly: boolean
+    value_usd: string
```

Comment:
let's make it consistent with Reports, change it to the usd_value, the same goes with the record type

Reply:
https://github.com/l2beat/l2beat/pull/1716/commits/fd67953de98a5af424c652917b773a5e4d8b3dec